english/amnesia/amnesia.md

```diff
@@ -1,4 +1,4 @@
-After eating marshmellows i can't get no sleep. Freaky noises make my
-skin creep. I can't get no sleep.
-
+After eating marshmellows i can't get no sleep. Freaky noises make my
+skin creep. I can't get no sleep.
+Then I open my mac and log on to udacity :)
```

Comment:
Removing the blank line removes the visual separation between the set-up and the option. Could you add it back?